ecl/regress/issue18564.ecl

```diff
@@ -0,0 +1,17 @@
+// LOADXML within a loop really doesn't work, but at least it no longer crashes.
```

Comment:
@ghalliday Why doesn't LOADXML work within a loop? A brief explanation may be useful.

Reply:
It doesn't work because of the way scoping of LOADXML interacts with the template variables.  I don't know exactly what happens.